packfiles.go

```diff
@@ -193,7 +200,7 @@ func getUnpackedObject(repo repository, hash plumbing.Hash) (o object.Object, er
 type repoObjectDecoder struct {
 	repo     string
 	packfile plumbing.Hash
-	decoder  *packfile.Decoder
+	decoder  *packfile.Packfile
```

Comment:
should we rename `decoder` to `packfile` then?

Reply:
... and remove/rename `packfile plumbing.Hash`?